Core/WumpusCore/Minigame.cs

```diff
@@ -0,0 +1,14 @@
+using WumpusCore.Trivia;
+
+namespace WumpusCore
+{
+    public interface Minigame
+    {
+        /// <summary>
+        /// Returns whether the player won
+        /// </summary>
+        /// <returns>True if won, False if lost, and null if game not over</returns>
+        GameResult reportResult();
```

Comment:
A little nit-picky but the returns comment here is a little confusing now that GameResult is an enum

Reply:
Fixed